### PR TITLE
Added new certs to BT v6

### PR DESCRIPTION
**Pull request by stechiu**
This PR has the new BT cert file. All flows have been tested in the Demo App in both Prod and Sandbox. The data is being sent to FPTI Explorer with the new certs. 

Related to this PR with new certs https://github.braintreeps.com/braintree/braintree-ios-certificate-generation/pull/2

### Checklist

- [x] Added a changelog entry

- @stechiu

### CODE REVIEW

File: Sources/BraintreeCore/BTAPIPinnedCertificates.swift [hunk 1]
```diff
@@ -8,152 +8,456 @@ import Foundation
 ///
 /// Do not update this file directly. If new certificates are needed, update the `certificates` directory
 /// in `braintree-ios-certificate-generation` and re-run the certificate generation script.
-class BTAPIPinnedCertificates {
+@objcMembers public class BTAPIPinnedCertificates: NSObject {
```
Comment (jaxdesmarais) [inline Sources/BraintreeCore/BTAPIPinnedCertificates.swift:11]: In [this PR](https://github.com/braintree/braintree_ios/pull/1052/files#diff-e0d221245e7631e0b1447a0720aee07ff3ee3b517f87dd68aaf1c377ec8f4e41) I updated this class/method to be non-public but didn't actually update the script. 🤦‍♂️ I'll put up a small PR to the cert script to make this change, but we will likely need to run the script again and update this PR once merged.

Comment (stechiu) [reply Sources/BraintreeCore/BTAPIPinnedCertificates.swift:11]: Ok no worries. Does your linked PR impact the merging of the new certs?

Comment (stechiu) [reply Sources/BraintreeCore/BTAPIPinnedCertificates.swift:11]: Ok I'll pull the latest from main and run it again!

File: Sources/BraintreeCore/BTAPIPinnedCertificates.swift [hunk 2]
```diff
@@ -10,150 +10,454 @@ import Foundation
 /// in `braintree-ios-certificate-generation` and re-run the certificate generation script.
 class BTAPIPinnedCertificates {
 
-    static func trustedCertificates() -> [NSData] {
+    public static func trustedCertificates() -> [NSData] {
```
Comment (jaxdesmarais) [inline Sources/BraintreeCore/BTAPIPinnedCertificates.swift:13]:
```suggestion
    static func trustedCertificates() -> [NSData] {
```

**Review by richherrera — APPROVED** [review]

**Review by jaxdesmarais — COMMENTED** [review]
Now that Ty's PR is merged, lets pull down main on the cert script and generate the certs 1 more time + update this PR with the result. Just want to be 110% sure what we have here accurately reflects what's on main since we were previously working off of Ty's branch.

**Comment by stechiu** [comment]
> Now that Ty's PR is merged, lets pull down main on the cert script and generate the certs 1 more time + update this PR with the result. Just want to be 110% sure what we have here accurately reflects what's on main since we were previously working off of Ty's branch.

Done ✅ I also updated the `CHANGELOG`. Let me know what you think of the wording

File: CHANGELOG.md [hunk 3]
```diff
@@ -2,6 +2,8 @@
 
 ## unreleased
 * Require Xcode 15.0+ (per [App Store requirements](https://developer.apple.com/news/?id=khzvxn8a))
+* Security
+  * Updated expiring pinned vendor SSL certificates
```
Comment (jaxdesmarais) [inline CHANGELOG.md:6]: Typically we used the header as the name of the module, so using security here may be confusing. I think we can remove this header and have this at the same level as the Xcode 15 requirement which will indicate it applies to the entire SDK.
```suggestion
* Updated expiring pinned vendor SSL certificates
```

**Review by sshropshire — APPROVED** [review]

File: CHANGELOG.md (hunk 3 above)
Comment (stechiu) [reply CHANGELOG.md:6]: Great thanks!

**Pull request merged**